### PR TITLE
DOC: changed formula in random.Generator.pareto doc #22701

### LINKED ISSUE #22701: DOC: error in the description of `numpy.random.pareto`
### Issue with current documentation:

The 'Notes' part of the documentation on Pareto distribution is wrong. The density in both pages reads as $p(x) = \frac{am^a}{x^{a+1}}$, but this is obviously not even a density, because the function $p(x)$ is not integrable for $a>0$.

https://numpy.org/doc/stable/reference/random/generated/numpy.random.Generator.pareto.html#numpy.random.Generator.pareto

https://numpy.org/doc/stable/reference/random/generated/numpy.random.pareto.html

### Idea or request for content:

_No response_

### PR DESCRIPTION
**Pull request by mkiffer**
Changed documentation according to the conversation in issue #22701 
Closes gh-22701

### CODE REVIEW

**Review by MuhammadNizamani — APPROVED** [review]
LGTM!

File: numpy/random/_generator.pyx [hunk 1]
```diff
@@ -2004,7 +2004,7 @@ cdef class Generator:
         -----
         The probability density for the Pareto distribution is
 
-        .. math:: p(x) = \\frac{am^a}{x^{a+1}}
+        .. math:: p(x) = \\frac{am^a}{{x+1}^{a+1}} , x \ge m
```
Comment (MatteoRaso) [inline numpy/random/_generator.pyx:2007]:
```suggestion
        .. math:: p(x) = \\frac{a}{{x+1}^{a+1}} , x \ge 0
```
As mentioned in the issue thread, there's no `m` in the true PDF, so it should be removed.

**Comment by MatteoRaso** [comment]
Thanks for the PR. Looking over the documentation for `numpy.random.pareto`, there's a lot more references to `m`, so it'd be nice if you removed those as well and replaced them with accurate descriptions.

File: numpy/random/_generator.pyx [hunk 2]
```diff
@@ -2004,7 +2004,7 @@ cdef class Generator:
...
 
         where :math:`a` is the shape and :math:`m` the scale.
```
Comment (MatteoRaso) [inline numpy/random/_generator.pyx:2009]:
```suggestion
        where :math:`a` is the shape.
```
As before.

**Comment by linus-md** [comment]
Hi @mkiffer there is not much left to do are you interested in retuning to this?

**Pull request merged**